decentraland/redux/reducers.js

```diff
@@ -83,13 +85,26 @@ function assetsForSale(state = { list: [], loadingInProgress: true }, action) {
 }
 
 function myAssets(state = { list: [] }, action) {
-  const { type, myAsset, myAssets, myAssetAndActionIds } = action;
+  const {
+    type,
+    myAsset,
+    myAssets,
+    myAssetAndActionIds,
+    myAssetAndStatus,
+    itemOrList,
+  } = action;
+
   switch (type) {
-    case ADD_TO_MY_ASSETS_LIST:
+    case PREPEND_TO_MY_ASSETS_LIST:
       return { ...state, list: [myAsset, ...state.list] };
-    case REMOVE_MY_ASSET_FROM_LIST: {
+    case APPEND_TO_MY_ASSETS_LIST: {
```

Comment:
@marcelomorgado Out of curiosity, do you think this `myAssets` reducer function is getting a little large to reason about? I'm wondering whether we can split it out into a few functions that each handle fewer different type cases.

(Comment made on somewhat arbitrary line)

Reply:
I agree. I've followed some tips from https://redux.js.org/recipes/structuring-reducers/refactoring-reducer-example seems better now? We can do the same for all reducers or even extract a file for each reducer.

Reply:
Okay, sounds good, thanks! I'll take a look.

Reply:
Yep, I like the change to this reducer. Resolving this!